### PR TITLE
Add ability to merge results of scan jobs to CLI

### PR DESCRIPTION
**Pull request by kholdaway**

### CODE REVIEW

File: qpc/report/merge.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import qpc.report as report
+from qpc.translation import _
+import qpc.messages as messages
+
+
+# pylint: disable=too-few-public-methods
+class ReportMergeCommand(CliCommand):
+    """Defines the report merge command.
+
+    This command merging scan job results into a
```
Comment (abaiken) [inline qpc/report/merge.py]: This command merging --> This command is for merging

File: qpc/report/merge.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            'jobs': self.args.scan_job_ids,
+        }
+
+    def _handle_response_success(self):
+        json_data = self.response.json()
+        print(_(messages.REPORT_SUCCESSFULLY_MERGED % json_data['id']))
+
+    def _handle_response_error(self):
+        json_data = self.response.json()
+        print(json_data['jobs'][0])
```
Comment (abaiken) [inline qpc/report/merge.py]: Should we be printing an error message here?

**Comment by kholdaway** [comment]
Updated man page

File: docs/source/man.rst [hunk 3]
```diff
@@ -513,6 +513,16 @@ For example, the raw facts of a scan that includes both network and vcenter sour
 
   Required. Path to a file location where the report data is saved.
 
+Merging Scan Job Results
+~~~~~~~~~~~~~~~~~~~~~~~~
+The ``qpc report merge`` command combines results from two or more scan jobs.
```
Comment (chambridge) [inline docs/source/man.rst]: `two or more scan jobs to create a report. The ``qpc report summary`` or ``qpc report detail` can be used to obtain the report.

Comment (kholdaway) [reply docs/source/man.rst]: @chambridge Fixed

**Pull request merged**